### PR TITLE
[#TX-28238]: updated the main path

### PR DESCRIPTION
**Pull request by lakshanban**
### changes 

* updated the main path

### ticket 

https://app.clickup.com/t/2704225/TX-28238

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "factory-girl",
   "description": "A factory library for Node.js and JavaScript inspired by factory_girl",
-  "main": "./src/index",
+  "main": "index.js",
```
Comment (thujee) [inline package.json]: shouldn't this be 
```suggestion
  "main": "./src/index.js",
```

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
       "transform-class-properties"
     ]
   },
-  "version": "5.0.3",
+  "version": "1.0.0",
```
Comment (thujee) [inline package.json]: let's not change the version
npm publish pipeline would take care of this
we'll need to include this package in the pipeline

File: package.json (hunk 1 above)
Comment (lakshanban) [reply package.json]: nope... we are publishing the build folder... so the file structure is :
Screenshot 2022-08-17 at 09 07 20

Comment (lakshanban) [reply package.json]: updated

File: package.json (hunk 2 above)
Comment (lakshanban) [reply package.json]: updated

**Review by thujee — APPROVED** [review]

**Pull request merged**